src/pages/mypage/BottomSection/index.tsx

```diff
@@ -0,0 +1,57 @@
+import * as styles from '@/pages/mypage/BottomSection/index.css';
+import Divider from '@/components/Divider';
+import Flex from '@/components/Flex';
+import Text from '@/components/Text';
+import { IcArrowRightSmallGray0432, IcArrowRightSmallGray0732 } from '@/assets/svg';
+import { MyPageProps } from '@/types/myPageTypes';
+
+// 클래스 등록 및 관리만 권한 필요
+const listData = [
+  { label: '내 정보 수정', id: 1 },
+  { label: '강사 등록', id: 2 },
+  { label: '클래스 등록', id: 3, isInstructorRequired: true },
+  { label: '클래스 관리', id: 4, isInstructorRequired: true, hasDivider: true },
+  { label: '자주 묻는 질문', id: 5 },
+  { label: '고객 센터', id: 6, hasDivider: true },
+  { label: '로그아웃', id: 7 },
+];
+
+const BottomSection = ({ userData }: { userData: MyPageProps }) => {
+  // 강사 권한을 가질 때는 null 이 아닌 number가 옴
+  const isInstructor = userData.lessonCount !== null;
+
+  return (
+    <Flex>
+      <ul className={styles.ulStyle}>
+        {listData.map((item) => {
+          const isDisabled = item.isInstructorRequired && !isInstructor;
+
+          return (
+            <ul key={item.id}>
+              <>
+                <li className={styles.listStyle}>
+                  <Text tag="b2" color={isDisabled ? 'gray4' : 'gray9'}>
+                    {item.label}
+                  </Text>
+                  {isDisabled ? (
+                    <IcArrowRightSmallGray0432 width={32} height={32} />
+                  ) : (
+                    <IcArrowRightSmallGray0732 width={32} height={32} />
+                  )}
+                </li>
+                {/* Divider가 필요한 항목에 추가 */}
+                {item.hasDivider && (
+                  <li className={styles.dividerStyle}>
+                    <Divider color="gray1" thickness={1} />
+                  </li>
+                )}
+              </>
+            </ul>
+          );
+        })}
+      </ul>
+    </Flex>
```

Comment:
```suggestion
    <Flex>
    <ul className={styles.ulStyle}>
      {listData.map((item) => {
        const isDisabled = item.isInstructorRequired && !isInstructor;

        return (
          <li key={item.id} className={styles.listStyle}>
            <Text tag="b2" color={isDisabled ? 'gray4' : 'gray9'}>
              {item.label}
            </Text>
            {isDisabled ? (
              <IcArrowRightSmallGray0432 width={32} height={32} />
            ) : (
              <IcArrowRightSmallGray0732 width={32} height={32} />
            )}
            {/* Divider가 필요한 항목에 추가 */}
            {item.hasDivider && (
              <div className={styles.dividerStyle}>
                <Divider color="gray1" thickness={1} />
              </div>
            )}
          </li>
        );
      })}
    </ul>
  </Flex>
```
현재 `<ul>`태그 내부에 `<ul>`태그가 중첩되어있는데  중첩된 `<ul>`은 `<li>`로 수정하고 `className={styles.listStyle}`을 적용해주는 것이 어떨까요? 현재 `<ul>`에 `<key>`값을 부여하고 있는데 그렇지 않아도 될거 같아요!

Reply:
반영했습니다!